AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/linked-participant-type.model.ts

```diff
@@ -0,0 +1,4 @@
+export enum LinkedParticipantType {
```

Comment:
I would use a union type here instead of an enum

Reply:
`export type LinkedParticipantType = 'Interpreter' | 'Interpretee'`

Reply:
deleted as was duplicate